includes/admin/class-llms-admin-review.php

```diff
@@ -75,13 +79,19 @@ public function admin_footer( $text ) {
 	/**
 	 * AJAX callback for dismissing the notice
 	 *
-	 * @return  void
-	 * @since   3.24.0
-	 * @version 3.24.0
+	 * @since 3.24.0
+	 * @since [version] Only users with `manager_lifterlms` caps can dismiss and added nonce verification.
+	 *              Use `llms_filter_input()` in favor of `filter_input()`.
```

Comment:
```suggestion
	 *               Use `llms_filter_input()` in favor of `filter_input()`.
```
If the next release will be 4.14.0.